public/app/features/dashboard/components/RowOptions/RowOptionsModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { Modal } from '@grafana/ui';
+import { css } from 'emotion';
+
+import { OnRowOptionsUpdate, RowOptionsForm } from './RowOptionsForm';
+
+export interface RowOptionsModalProps {
+  title: string | null;
+  repeat: string | null;
+  onDismiss: () => void;
+  onUpdate: OnRowOptionsUpdate;
+}
+
+export const RowOptionsModal: FC<RowOptionsModalProps> = ({ repeat, title, onDismiss, onUpdate }) => {
+  return (
+    <Modal
+      isOpen={true}
+      title="Row Options"
+      icon="copy"
+      onDismiss={onDismiss}
+      className={css`
```

Comment:
nit: in this case I would create a styles constant that holds the styling. Not that it is super important in this scenario but with this approach it makes the probability of continue to add styles in the same way lower.

Reply:
Splendid catch and I agree!